plugins/ruby-foreman-azure-rm/debian/rules

```diff
@@ -1,17 +1,9 @@
 #!/usr/bin/make -f
 
 PLUGIN = foreman_azure_rm
+PLUGIN_HAS_ASSETS = true
+PLUGIN_HAS_WEBPACK = false
+PLUGIN_HAS_PACKAGE_JSON = false
+PLUGIN_HAS_API = true
```

Comment:
This is redundant since c72f55a4ebe6f346fb69ad9e0bc5061e7810d664 but it looks like the repo wasn't entirely purged of it.

Reply:
Correct, I have it already removed in my local branch, but need https://github.com/theforeman/foreman-packaging/pull/8256 fixed first :see_no_evil: